SETUP_ORACLE.md

```diff
@@ -68,7 +68,28 @@ You need to have Docker installed and running. You also need to be able to [run
 
 1. From this repository, do `make oracle-database` and wait to see *DATABASE IS READY TO USE!*.
 
-2. Finally initialize the database with some seed data by running: `DATABASE_URL="oracle-enhanced://rails:railspass@127.0.0.1:1521/systempdb" ORACLE_SYSTEM_PASSWORD=threescalepass NLS_LANG=AMERICAN_AMERICA.UTF8 USER_PASSWORD=123456 MASTER_PASSWORD=123456 MASTER_ACCESS_TOKEN=token bundle exec rake db:drop db:create db:setup`
+    1. This will create a new user to stablish regular connections with the Oracle Database. For that, Oracle's SYSTEM user must be used.
+    2. Alternatively, the `ORACLE_SYSTEM_PASSWORD` ENV variable might be omitted and, in this case, a valid user must be provided.
```

Comment:
It took me a lot of time to figure out what this means, so in the step 3 you are using ORACLE_SYSTEM_PASSWORD, but if you want to omit it, then you need to connect to the DB and run the script bellow. I agree with @akostadinov that this should be reordered. IMO it should be like this:

1. make oracle-databse...
2. initialize the database...
    - [Optional]  Alternatively, the `ORACLE_SYSTEM_PASSWORD`...

Reply:
I accepted @akostadinov's suggestions and now it looks clearer. Could you please check if it's OK now?